omnibus/cookbooks/omnibus-supermarket/templates/nginx.conf.erb

```diff
@@ -35,8 +35,9 @@ http {
   <% unless @nginx['disable_access_log'] -%>
   access_log	<%= @nginx['log_dir'] %>/access.log supermarket;
   <% end %>
-  
+
   server_tokens off;
+  add_header X-Clacks-Overhead "GNU Terry Pratchett";
```

Comment:
@robbkidd This one really confused me when I was testing security headers until I found this PR